src/posts/introducing-inference-data/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Better Bayesian Workflows with InferenceData and Xarray'
+date: '2022-12-17'
+authors:
+  - name: Ravin Kumar
+    github: canyon289
+  - name: Oriol Abril-Pla
+    github: OriolAbril
+  - name: Osvaldo Martin
+    github: aloctavodia
+summary: "Xarray provides the core data model for ArviZ's InfrerenceData structure"
+---
+
+_TLDR: Xarray is the core of the `az.InferenceData` object. [ArviZ InferenceData](https://python.arviz.org/en/stable/api/generated/arviz.InferenceData.html) simplifies the Bayesian workflow, facilitates reproducibility, and enables interoperability between different Probabilistic Programming languages._
+
+```python
+# Load the saved results of a bayesian analysis from disk
+import arviz as az
+
+data = az.load_arviz_data("centered_eight")
+data
```

Comment:
Can we show the repr for this?

Reply:
The repr is not very informative, it will look like:

```
Inference data with groups:
	> posterior
	> posterior_predictive
	> log_likelihood
	> sample_stats
	> prior
	> prior_predictive
	> observed_data
	> constant_data
```

If possible though, the html_repr is much more informative (built on top of the dataset one), it is the output of the 2nd code cell in https://python.arviz.org/en/stable/getting_started/WorkingWithInferenceData.html.

Reply:
we should be able to put the html repr in the post.

Reply:
@OriolAbril I think we could show both the inferencedata repr, and then the repr of one of the groups, which is just an xarray repr. What do you think? Would that better highlight now the two are related?

![image](https://user-images.githubusercontent.com/7213793/230734449-ea2a7f5b-4521-40ab-8509-c6c8dc5f8c6f.png)

Reply:
I think it is simpler to use the inferencedata html repr directly, omce any of the groups is expanded it becomes clear and explicit they are xarray.Dataset objects

Reply:
Really want to get this over the line. What does it take? Should I copy the html from here?

https://python.arviz.org/en/stable/getting_started/XarrayforArviZ.html#an-introduction-to-each

is it a matter of copying it from the static notebook in github, or do I need to do it from jupyter?